mu4/notation/internal/notationactioncontroller.cpp

```diff
@@ -217,3 +218,14 @@ void NotationActionController::openPageStyle()
 {
     interactive()->open("musescore://notation/style");
 }
+
+void NotationActionController::openStaffProperties()
+{
+    auto interaction = currentNotationInteraction();
+    if (!interaction) {
+        return;
+    }
+
+    int staffIdx = interaction->selection()->firstStaffIndex();
+    interactive()->open("musescore://notation/staffproperties?sync=false&staffIdx=" + QString::number(staffIdx).toStdString());
```

Comment:
sync=false

Is this not-modal dialog?

Reply:
to close the context menu before opening the dialog

Reply:
fixed